packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -264,12 +300,12 @@ try {
   const vscode = acquireVsCodeApi();
 
   window.onload = () => {
-    initDataviz(vscode);
-    console.log("ready");
+    console.log("post webviewStyleRequest");
```

Comment:
I would keep it `ready` here? I am not sure how this is `post webviewStyleRequest`

Reply:
This was debug code that I forgot to remove, it got switched around and ended like this. My bad